ci/tasks/scripts/with-docker.sh

```diff
@@ -9,7 +11,7 @@ function load_layers() {
 
   # don't do anything if we don't have cache
   [[ ! -d 'docker-cache' ]] && return 0;
-  [[ -z "$(ls docker-cache)" ]] && return 0;
+  [[ -z "$(ls docker-cache/*/image.tar)" ]] && return 0;
```

Comment:
This will stop a failing `ls` from aborting your script
```suggestion
  [[ -z "$(ls docker-cache/*/image.tar || :)" ]] && return 0;
```

Reply:
Given that we're using `&&` within the statement, this isn't needed

Reply:
Good point